Netable/Netable/Netable.swift

```diff
@@ -189,9 +233,16 @@ public actor Netable {
 
         let retryConfiguration = self.retryConfiguration
 
+        if let mocked = mockedUrl {
+            let data = try Data(contentsOf: mocked)
+            let decoded = try await request.decode(data, defaultDecodingStrategy: self.config.jsonDecodingStrategy)
+            let finalizedResult = try await request.finalize(raw: decoded)
+            return finalizedResult
```

Comment:
Not ideal that the decode / finalization logic is duplicated between mocked and real requests, if we ever tweaked this (say added another phase to request finalization), there'd now be a risk of having the two places this is done get subtly out of sync.